lircd.te

```diff
@@ -40,6 +40,8 @@ dev_filetrans(lircd_t, lircd_var_run_t, sock_file)
 
 kernel_request_load_module(lircd_t)
 
+auth_read_passwd(lircd_t)
```

Comment:
Changes are OK. We just need to follow StyleGuide. See 

https://github.com/TresysTechnology/refpolicy/wiki/StyleGuide

and _Local policy rules_ section. It means you need to call

         auth_read_passwd(lircd_t)

after 

        term_use_unallocated_ttys(lircd_t)

Thank you.